hipchat.go

```diff
@@ -77,6 +77,14 @@ type ErrorResponse struct {
 
 type Client struct {
 	AuthToken string
+	BaseURL   string
```

Comment:
change the name of the `baseURL` const to `defaultBaseURL` in lieu of this change.